pkg/asset/cluster/aws/aws.go

```diff
@@ -95,3 +113,11 @@ func tagSharedVPCResources(ctx context.Context, clusterID string, installConfig
 func sharedTag(clusterID string) (string, string) {
 	return fmt.Sprintf("kubernetes.io/cluster/%s", clusterID), "shared"
 }
+
+func publicElbTag() (string, string) {
+	return "kubernetes.io/role/elb", ""
```

Comment:
nit: should we consider making these constants as they have no value associated with the key?